.github/workflows/main.yml

```diff
@@ -15,12 +15,12 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  test_and_doc:
+  test:
     # The type of runner that the job will run on
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest,] # windows-latest]
+        os: [ubuntu-latest,]
```

Comment:
only test on ubuntu, because we only need to run it on debian on raspberry. Getting virtual audio devices to run the tests in the pipeline proved to be a time intensive hassle, and while I eventually did get it to run on ubuntu and macos, I never managed on windows, and decided to skip both eventually.